### DISCUSSION TITLE
Conexión Xdrp+ en android y Dexcom G6

### DISCUSSION THREAD

**Discussion by duvimontes** [NightscoutFoundation/xDrip#3741] [Q&A]
Hola, 
Me presento, soy Duvimontes, nuevo usuario.

Actualmente estoy usando por primera vez Xdrip+ junto con mi sensor y trasmisor Dexcom g6. He seguido todos los pasos que se incluyen en las guías, y no consigo  que la aplicación enlace con mi trasmisor dexcom. 

Incluyo toda la información que se solicita previamente.
- Dispositivo: Dexcom G6 + trasmisor.
- Modelo móvil: Samsung S24 con sistema operativo Android 14.
- Incluyo capturas de Estado del sistema y de Captura de fuente de datos.

![Screenshot_20241029_103254_xDrip+](https://github.com/user-attachments/assets/d2933477-2c46-4de7-af37-45b67a367edb)
![Screenshot_20241029_103628_xDrip+](https://github.com/user-attachments/assets/045e09ed-e363-477b-a039-862658d1779a)
![Screenshot_20241029_095331_xDrip+](https://github.com/user-attachments/assets/d1d3620b-5132-4759-99de-6b7202c7cbbf)
![Screenshot_20241029_095345_xDrip+](https://github.com/user-attachments/assets/cfb5c318-68fb-48be-b0de-81c57d8973ba)
![Screenshot_20241029_095359_xDrip+](https://github.com/user-attachments/assets/596e48c6-1df8-4ccf-b183-d3c9412107e4)

Disculpad las molestias, he buscado temas similares, pero con las soluciones que he visto, no he podido solventar mi problema de conexión. 
Gracias.

**Comment by Navid200** [comment]
Looking at the screenshots, it looks like you have Dexcom app installed.
If you need to use Dexcom app, you cannot use xDrip as a collector.
However, you can use xDrip in Companion App mode:
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Comment by duvimontes** [comment]
Hola,

Muchas gracias por la ayuda. Así es, sigo usando la aplicación Dexcom.

He seguido los pasos que aparecen en la guía que me adjuntas, pero ahora no
sé como establecer la relación con Xdrip para que esta aparezca de forma
complementaria.

En la sección "Acceso a notificaciones" me aparece como en la imagen. No sé
si me falta algo por configurar. De nuevo pido disculpas, estoy algo
perdido.

Muchas gracias de nuevo y disculpa.

**Reply by Navid200** [reply]
Have you given notification access to xDrip?
![Screenshot_20241029-151233](https://github.com/user-attachments/assets/cea17cfd-5323-4abe-a8cc-6496106ccbcc)

**Comment by duvimontes** [comment]
Si, he dado todos los permisos.

Al activar el Companion App mode, sigo los pasos de la guía, pero nunca
llega a indicar mi glucosa.

Debo registrar de alguna forma que debe obtener los datos de medición del
sensor G6 o de la aplicación g6 no?

**Reply by Navid200** [reply]
Does Dexcom app show a new notification once every 5 minutes?
If yes, can you post a screenshot?

**Comment by duvimontes** [comment]
Hola,

Conseguí que los datos aparecieran em XDrip+. Estoy usando sensores
Freestyle 3 y ha sido fácil configurarlo.

También he podido llevar las mediciones a  Garmin. No he usado base de
datos Nightscout si no un widget para mediante bluetooth enviar las
mediciones a mi reloj.

Muchas gracias por todo y disculpa las molestias creadas.

Un cordial saludo!
